src/store/rootReducer.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import { authorsReducer } from './authors/reducer.js';
+import { coursesReducer } from './courses/reducer.js';
+import { userReducer } from './user/reducer.js';
+
+export const rootReducer = combineReducers({
+	courses: coursesReducer,
+	authors: authorsReducer,
+	user: userReducer,
+});
+
+export const store = configureStore({ reducer: rootReducer });
```

Comment:
this shouldn't be here, you have this piece of code in your index.js from this folder